docs/source/resources.rst

```diff
@@ -27,6 +27,18 @@ This will help with:
 * Highlighting where there are known erroneous entries for each field
 * Recommending remedial actions for erroneous entries within each field
 
+An :download:`additional spreadsheet <docs/NCCID_site_specific_units.xlsx>` contains
+information on submitting sites that use measurement units for various clinical variable fields
+that is different from the overview of the above schema description. This will help with:
+
+* Transforming clinical measurements on the same range
+
+Please note, that this sheet contains only a subset of potential submitting centres, and a
+subset of clinical variables, as it was possible to collect. For some fields you might
```

Comment:
Please note that this sheet contains only a subset of potential submitting centres, and a
subset of clinical variables. Furthermore, responses for these submitting centres may only represent a subset of their hospital sites, where units might vary from hospital to hospital. The information presented here should only serve as a guide, and always be checked against values to ensure they are sensible. For some fields you might have to make decisions on data cleaning and potentially drop values that you think are outside the plausible range or entered incorrectly.